dags/veda_data_pipeline/groups/discover_group.py

```diff
@@ -22,7 +22,7 @@ def discover_from_s3_task(ti, event={}, **kwargs):
         **ti.dag_run.conf,
     }
     last_successful_execution = kwargs.get("prev_start_date_success")
-    if last_successful_execution:
+    if event.get("schedule") and last_successful_execution:
         config["last_successful_execution"] = last_successful_execution.isoformat()
```

Comment:
We only want to add the `last_successful_execution` field to the event if it is a scheduled delivery (which is determined by if there's a "schedule" field in the incoming `event`.

This right now is causing "no files discovered" error, because the files are there since the `veda_discover` dag was last run.

This fixes this.